Sources/CorePayments/Networking/TrackingEventsAPI.swift

```diff
@@ -4,13 +4,29 @@ import Foundation
 ///
 /// Details on this PayPal API can be found in PPaaS under Infrastructure > Experimentation > Tracking Events > v1.
 class TrackingEventsAPI {
+        
+    // MARK: - Internal Properties
+
+    private var apiClient: APIClient
+    var coreConfig: CoreConfig // exposed for testing
+
+    // MARK: - Initializer
+    
+    init(coreConfig merchantConfig: CoreConfig) {
```

Comment:
It looks like the other APIs don't have an internal parameter name for `coreConfig`. Do we see an advantage to doing it here vs the other API layers?

Reply:
Good eye - so for the `TrackingEventsAPI` we're basically overriding the merchant-config to always be `.live` since FPTI doesn't support the traditional PP sandbox URL. 

How can we make that more clear? Maybe we can move the comment on line 16 around somewhere else?

Reply:
Gotcha, I was more curious around why line 17 couldn't be:
```swift
        self.coreConfig = CoreConfig(clientID: coreConfig.clientID, environment: .live)
```
Then the init can match the other API class inits `init(coreConfi: CoreConfig)`

Reply:
It could be! I just thought using the internal var name of `merchantConfig` might make it more clear that we're overriding it to set our own `coreConfig`. The public API of the init matches the others.

But I'm happy to change it back if this isn't clear

Reply:
The whole init setup for this class is ugly. We technically don't need custom inits at all (like it was before). But we _do_ need to set the APIClient & access to the coreConfig if we want to properly unit test

That said - totally down for your opinion on what is easiest to read here!

Reply:
Gotcha - imo with your inline comment it makes it clear that we aren't using the environment from the config. But I don't feel super strongly if you feel like the internal parameter name makes it more clear in addition to the comment you have.

Reply:
2 cents: I do like the internal name. It prevents variable shadowing, and technically the signature of the constructor is still `init(coreConfig: CoreConfig)` externally. The internal parameter name won't leak outside of this file.